pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -1202,6 +1255,9 @@ const New = () => {
 									</div>
 								</div>
 							</div>
+							<Button onClick={upload} title={'Upload'}>
+								<h1>UPLOAD TO ARWEAVE. MUST BE DONE BEFORE ADDING PROPOSAL!!!!</h1>
```

Comment:
@nicetransition It takes a few seconds for the arweave upload to succeed and return the link, so I made this a separate step. Figured you would have a better way to chain the upload and use of the link in the proposal, but didn't want to spend too much time on fumbling my way through it.